docs/community/related_projects.rst

```diff
@@ -0,0 +1,78 @@
+.. _relatedprojects :
+
+Related Projects
+================
+
+On this page we list a number of projects that are
+affiliated with or makes use of QCoDeS in some way or another.
+
+If you would like to get your project featured here please submit a
+pull request adding your project to this page.
+The page is located at ``docs\community\related_projects.rst`` within the QCoDeS
+repository.
+
+
+QCoDeS contrib drivers
+----------------------
+
+QCoDeS contrib drivers is a collection of drivers for many
+different instruments developed by the community. If you have
+developed a driver that your wish to share with the community
+this is the ideal place for it.
+
+Broadbean can be installed with pip from PyPi
+with ``pip install qcodes_contrib_drivers``
+The source code is hosted on Github
+`here <https://github.com/qcodes/qcodes_contrib_drivers>`__
+and the documentation can be found
+`here <https://qcodes.github.io/Qcodes_contrib_drivers/>`__
+
+
+Broadbean
```

Comment:
shall we also mention https://github.com/QCoDeS/lomentum ? or only when it becomes published on pypi etc etc ?

Reply:
Even the repo is private at the moment so I would say no